src/open_inwoner/templates/pages/cases/list.html

```diff
@@ -15,9 +15,11 @@ <h2 class="h2" id="cases">{{ page_title }} ({{ paginator.count }})</h2>
                 {% render_column start=forloop.counter_0|multiply:4 span=4 %}
                     {% render_card compact=True stretch=True title=case.identificatie %}
                         {% render_list %}
+                            <a href="/accounts/cases/{{ case.uuid }}/status/" class="case-list">
```

Comment:
Don't build urls as strings, use the `{% url %}` tag to reverse it from the router: https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#url

I think it'd be like `{% url 'accounts:case_status' object_id=case.uuid %}`